stable/prometheus-blackbox-exporter/templates/deployment.yaml

```diff
@@ -51,8 +51,9 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           securityContext:
+            readOnlyRootFilesystem: true
```

Comment:
mak this configurable and set the default value

Reply:
Thanks @cpanato , PR updated